docs/Troubleshooting/Configuration.md

```diff
@@ -68,6 +68,14 @@ Here is an example of the `docker-compose` installation, the idea applies to oth
     command: --innodb-buffer-pool-size=200M
 ```
 
+### Repositories don't show up to select for data scopes in GitHub connections
+See issue [#6038](https://github.com/apache/incubator-devlake/issues/6038)
+
+First you should check whether you access token has necessary permissions, if permissions are correct. Please make sure that after creating the token you have authorized it using SSO if your account are protected by organization SAML enforcement.
```

Comment:
1. ... whether your access token has the necessary permissions.
2. If permission are correct, please ...